tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -19,13 +17,13 @@ const randomNumber = (max) => Math.floor(Math.random() * max);
  * @param {string} targets[].counter
  */
 const expectTargets = async (targets) => {
-  expect(await element.all(by.css(`#target-aggregates-list li`)).count()).toEqual(targets.length);
+  expect(await (await analytics.aggregateList()).length).to.equal(targets.length);
 
   const expectTarget = async (target) => {
-    const lineItem = () => element(by.css(`#target-aggregates-list li[data-record-id=${target.id}]`));
-    expect(await lineItem().isPresent()).toBe(true);
-    expect(await lineItem().element(by.css('h4')).getText()).toEqual(target.title);
-    expect(await lineItem().element(by.css('.aggregate-status span')).getText()).toEqual(target.counter);
+    const lineItem = () => $(`#target-aggregates-list li[data-record-id=${target.id}]`);
```

Comment:
Can you put the selectors definitions into the corresponding page-object file please?